core/common/src/rpc/web/ITwinCloudRpcProtocol.ts

```diff
@@ -0,0 +1,148 @@
+  public getOperationFromPath(path: string): SerializedRpcOperation {
+    const url = new URL(path, "https://localhost/");
+    const components = url.pathname.split("/");
+
+    const operationComponent = components.slice(-1)[0];
+    const encodedRequest = url.searchParams.get("parameters") || "";
+
+    const firstHyphen = operationComponent.indexOf("-");
+    const lastHyphen = operationComponent.lastIndexOf("-");
+    const interfaceDefinition = operationComponent.slice(0, firstHyphen);
+    const interfaceVersion = operationComponent.slice(firstHyphen + 1, lastHyphen);
+    const operationName = operationComponent.slice(lastHyphen + 1);
+
+    return { interfaceDefinition, operationName, interfaceVersion, encodedRequest };
+  }
+
+  /** Supplies the OpenAPI-compatible URI path for an RPC operation. */
+  public supplyPathForOperation(operation: RpcOperation, request: RpcRequest | undefined) {
+    const prefix = this.pathPrefix;
+    const operationName = operation.operationName;
+
+    let contextId: string = "";
+    let iModelId: string = "";
+    let routeChangeSetId: string | undefined;
+    /* Note: The changeSetId field is omitted in the route in the case of ReadWrite connections since the connection is generally expected to be at the
+     * latest version and not some specific changeSet. Also, for the first version (before any changeSets), the changeSetId in the route is arbitrarily
+     * set to "0" instead of an empty string, since the latter is more un-intuitive for a route. However, in all other use cases, including the changeSetId
+     * held by the IModelRpcProps itself, the changeSetId of "" (i.e., empty string) signifies the first version - this is more intuitive and retains
+     * compatibility with the majority of use cases. */
+
+    if (request === undefined) {
+      contextId = "{contextId}";
+      iModelId = "{iModelId}";
+      routeChangeSetId = "{changeSetId}";
+    } else {
+      let token = operation.policy.token(request) || RpcOperation.fallbackToken;
+
+      if (!token || !token.iModelId) {
+        if (RpcConfiguration.disableRoutingValidation) {
+          token = { key: "" };
+        } else {
+          throw new IModelError(BentleyStatus.ERROR, "Invalid iModelToken for RPC operation request", Logger.logError, CommonLoggerCategory.RpcInterfaceFrontend);
+        }
+      }
+
+      contextId = encodeURIComponent(token.contextId || "");
+      iModelId = encodeURIComponent(token.iModelId!);
+      routeChangeSetId = token.changeSetId || "0";
+    }
+
+    return `${prefix}/visualization/context/${contextId}/imodel/${iModelId}${!!routeChangeSetId ? `/changeset/${routeChangeSetId}` : ""}/${operationName}`;
+  }
+
+  /**
+   * Inflates the IModelRpcProps from the URL path for each request on the backend.
+   * @note This function updates the IModelRpcProps value supplied in the request body.
+   */
+  public inflateToken(tokenFromBody: IModelRpcProps, request: SerializedRpcRequest): IModelRpcProps {
```

Comment:
I deleted "mode" from this function in BentleyCloudRpcProtocol, but if backends are going to be seeing the same requests as before (with the additional "Accept" header) should I be leaving all backend related methods unchanged?